src/repository/db/db_repository.go

```diff
@@ -0,0 +1,260 @@
+package db
+
+import (
+	"encoding/csv"
+	"fmt"
+	"github.com/spf13/viper"
+	"golang-bootcamp-2020/domain/model"
+	_errors "golang-bootcamp-2020/utils/error"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var (
+	isCsvFetched  = false
+	charactersMap map[string]*model.Character
+)
+
+const (
+	chId = iota
```

Comment:
By convention (most lints even show a warning) the Id fields should be renamed as ID
And Url to URL